src/cmds/scripts/pbs_postinstall.in

```diff
@@ -55,11 +55,6 @@ is_cray_xt() {
 }
 
 check_switch() {
-	adapters=`ntblstatus 2> /dev/null | awk '/^Adapter/ {print $2}' | uniq`
```

Comment:
Here also we don't nee this function anymore as it was used only under AIX section which is removed.